src/libs/antares/study/parts/short-term-storage/properties.h

```diff
@@ -52,20 +52,20 @@ unsigned int groupIndex(Group group);
 class Properties
 {
 public:
-    bool validate();
+    bool validate(bool simplexIsWeek);
     bool loadKey(const IniFile::Property* p);
     // Not optional   Injection nominal capacity, >= 0
     std::optional<double> injectionCapacity;
     // Not optional   Withdrawal nominal capacity, >= 0
     std::optional<double> withdrawalCapacity;
     // Not optional   Reservoir capacity in MWh, >= 0
     std::optional<double> capacity;
-    // Initial level, <= capacity
+    // Initial level, between 0 and 1
     std::optional<double> initialLevel;
     // Efficiency factor between 0 and 1
     double efficiencyFactor = 1;
-    // Cycle duration, 1 <= cycleDuration <= 168
-    unsigned int cycleDuration;
+    // Not optional   Cycle duration, 1 <= storagecycle <= 168
+    std::optional<unsigned int> storagecycle;
     // Used to sort outputs
     Group group = Group::Other1;
```

Comment:
map **ST_STORAGE_PROPERTY_GROUP_ENUM** is used only inside **Properties** class, we should make it **private**.